benchmarks/torchbench_model.py

```diff
@@ -228,9 +228,12 @@ def load_benchmark(self):
     # workaround "RuntimeError: not allowed to set torch.backends.cudnn flags"
     # torch.backends.__allow_nonbracketed_mutation_flag = True
 
+    # torchbench uses `xla` as device instead of `tpu`
+    if device := self.benchmark_experiment.accelerator == 'tpu':
+      device = 'xla'
```

Comment:
Since `tpu` devices may have more than one index, shouldn't we use `str(self.benchmark_experiment.get_device())`, instead?

Reply:
Thanks for catching that! Update to `get_device()` method.